index.html

```diff
@@ -1,14 +1,459 @@
+            crafted<br />
+            by its GDG organizers to fit the learning needs and interests of
+            their<br />
+            local developer community, with a strong focus on knowledge
+            exchange,<br />
+            networking, and learning about Google developer technologies.
+          </p>
+          <div class="m-5 pb-5">
+            <p>
+              Join a local group and meet local developers with similar
+              interests.
+            </p>
+            <a class="btn-lg text-dark fw-semibold" href="#" role="button"
+              >Find a Google Developer Group near you ></a
+            >
+          </div>
+        </section>
+      </div>
+      <div class="d-flex justify-content-center border border-2">
+        <section class="text-center">
+          <div class="main-title position-relative m-5">
+            <h2 class="fw-semibold">Google Developer Group</h2>
+          </div>
+          <p class="d-block d-md-none white-bg m-5 p-4 ps-5 pe-5 text-wrap">
+            Google Developer Groups represents a network of technical
+            communities that offer opportunities for professional development
+            and connection.
+          </p>
+          <p class="d-none d-md-block white-bg m-5 p-4 ps-5 pe-5 text-wrap">
+            Google Developer Groups represents a network of technical
+            communities that offer <br />opportunities for professional
+            development and connection. Those who join can learn about the
+            latest Google<br />
+            tools and technologies, and can meet with local developers with
+            similar interests.
+          </p>
+          <div class="m-5">
+            <img
+              class="w-25 border border-2"
+              src="./visuals/gdg-member-badge.png"
+              alt=""
+            />
+          </div>
+        </section>
+      </div>
+      <div class="d-flex justify-content-center">
+        <section class="text-center">
+          <div class="main-title position-relative m-5">
+            <h2 class="fw-semibold">Earnable badges</h2>
+          </div>
+          <p class="white-bg mb-5 pb-4 ps-5 pe-5 text-wrap">
+            Earn these badges when you visit and register for 2022 DevFest
+            event.
+          </p>
+          <div
+            class="d-flex d-md-none m-5 flex-column justify-content-center align-items-center"
+          >
+            <img
+              class="w-50 m-1 border border-2"
+              src="./visuals/badge-visit.svg"
+              alt=""
+            />
+            <img
+              class="w-50 m-1 border border-2"
+              src="./visuals/badge-register.svg"
+              alt=""
+            />
+          </div>
+          <div class="d-none d-md-flex m-5 justify-content-center">
+            <img
+              class="w-50 m-1 border border-2"
+              src="./visuals/badge-visit.svg"

```

Comment:
As the previous reviewer noted, please update the styling of the images in the past events section of the about page to match the behance design better🙏. It is one of the project requirements for your design to be almost identical to the behance design. Please update your styling to be a background image with an orange overlay and text centered in the middle.

Your design:
<img width="534" alt="image" src="https://user-images.githubusercontent.com/37116322/225148220-6068f7bf-3643-4119-8ae0-54acef39a529.png">


Behance design:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/37116322/225148321-0681b7c0-4835-41c8-ae6c-ac363dea6f29.png">